packages/devtools/lib/src/utils.dart

```diff
@@ -50,7 +50,9 @@ String escape(String text) => text == null ? '' : htmlEscape.convert(text);
 
 final NumberFormat nf = NumberFormat.decimalPattern();
 
-String percent(double d) => '${(d * 100).toStringAsFixed(1)}%';
+String percent0(double d) => '${(d * 100).toStringAsFixed(0)}%';
+
+String percent1(double d) => '${(d * 100).toStringAsFixed(1)}%';
```

Comment:
for your use case rounding to a specific number of digits of precision seems odd. I would just display the double value as a %. That way there won't be unexpected rounding and output should be smaller for simple cases. For example:
0.8 should show up as 80%